lib/AppInfo/Application.php

```diff
@@ -41,6 +41,7 @@ class Application extends App implements IBootstrap {
 	public const DEFAULT_TRANSCRIPTION_MODEL_ID = 'whisper-1';
 	public const DEFAULT_DEFAULT_IMAGE_SIZE = '512x512';
 	public const MAX_GENERATION_IDLE_TIME = 60 * 60 * 24 * 10;
+	public const DEFAULT_CHUNK_SIZE = 0; // no chunking
```

Comment:
I wonder if we should add a sensible default here? GPT3.5 has a context size of 16385 for example. We could set this to 10k? What do you think?

Reply:
(See below about discrepancy between character count and token count.)